.github/workflows/ci.yml

```diff
@@ -31,17 +31,19 @@ jobs:
       run: pip install -r requirements/pip.txt
 
     - name: Install Dependencies
-      run:
+      run: |
         pip install setuptools
         pip install -r requirements/ci.txt
 
     - name: Run Tests
       env:
         TOXENV: ${{ matrix.toxenv }}
+      # Sleep is needed to give elasticsearch enough time to startup.
+      # Longer term we should switch to the upstream ES github action to start up the server
       run: sleep 10 && tox
 
     - name: Run Coverage
-      if: matrix.python-version == '3.8' && matrix.toxenv=='django42'
+      if: matrix.python-version == '3.12' && matrix.toxenv=='django42'
```

Comment:
If we support 3.11 should we also run coverage against it?

Reply:
We've been running coverage and style only against the latest version which seems to be sufficient and hasn't caused any major issues.